LISA/Executor.cpp

```diff
@@ -75,11 +51,27 @@ std::string generateHandle()
 
 enum ReturnCodes {
     ERROR_NONE = 0, //Core::ERROR_NONE,
+    ERROR_GENERAL = 1,
     ERROR_WRONG_PARAMS = 1001,
     ERROR_TOO_MANY_REQUESTS = 1002,
     ERROR_ALREADY_INSTALLED = 1003,
 };
 
+uint32_t Executor::Configure(const std::string& dbPath)
+{
+    auto result{ERROR_GENERAL};
+    try {
+        handleDirectories();
+        initializeDataBase(dbPath);
+        result = ERROR_NONE;
```

Comment:
Maybe set this at the beginning, so there will be no additional assignment. We return ERROR_GENERAL only when exception is fired.

Reply:
hm, yeah, I can change it in the next patch